pkg/config/setup/system_probe.go

```diff
@@ -402,6 +402,7 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Config) {
 	// Discovery config
 	cfg.BindEnvAndSetDefault(join(discoveryNS, "enabled"), false)
 	cfg.BindEnvAndSetDefault(join(discoveryNS, "cpu_usage_update_delay"), "60s")
+	cfg.BindEnvAndSetDefault(join(discoveryNS, "ignore_comms"), "")
```

Comment:
Why don't we put the default values we have hard coded in the configuration as the default value?
Find a better name than `ignore_comms`. That is a customer facing attribute, it should be clear for the user what is it doing

Reply:
I've changed it to save list of default commnds